ods_ci/tests/Tests/0700__feature_store/test-run-feast-operator-e2e.robot

```diff
@@ -0,0 +1,46 @@
+*** Settings ***
+Documentation     Feast Operator E2E tests - https://github.com/opendatahub-io/feast/tree/master/infra/feast-operator/test/e2e
+Suite Setup       Prepare Feast E2E Test Suite
+Suite Teardown    Teardown Feast E2E Test Suite
+Library           Process
+Resource          ../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+Resource          ../../../tests/Resources/Common.robot
+Resource          ../../Resources/Page/FeatureStore/FeatureStore.resource
+
+
+*** Variables ***
+${FEATURE-STORE-RELEASE-TAG}             adjustment-release-v0.48.1
+${FEATURE-STORE_DIR}                     feature-store
+${FEATURE-STORE_REPO_URL}                %{FEATURE-STORE_REPO_URL=https://github.com/opendatahub-io/feast.git}
+
+
+*** Test Cases ***
+Run runTestDeploySimpleCRFunc test
+    [Documentation]    Run Go E2E test: runTestDeploySimpleCRFunc
+    [Tags]  smoke
+    ...     FeatureStore
+    ...     RHOAIENG-14799
+    Run Feast Operator E2E Test    TesDefaultFeastCR
+
+Run runTestWithRemoteRegistryFunction test
+    [Documentation]    Run Go E2E test: runTestWithRemoteRegistryFunction
+    [Tags]  smoke
+    ...     FeatureStore
+    ...     RHOAIENG-14799
+    Run Feast Operator E2E Test    TestRemoteRegistryFeastCR
+
+
+*** Keywords ***
+Prepare Feast E2E Test Suite
+    [Documentation]    Prepare Feast E2E Test Suite
+    Skip If Component Is Not Enabled     feastoperator
```

Comment:
Remove the skip condition. Since we're now enabling the Feast Operator component as part of the prerequisites, the test will always be skipped otherwise

Reply:
Fixed in https://github.com/red-hat-data-services/ods-ci/pull/2408/commits/1116e131cec5d5fbe8cc3c71e4c062e0269e3911